Plugins/NativeXr/Source/NativeXr.cpp

```diff
@@ -463,15 +463,15 @@ namespace Babylon
 
                     m_sessionState = std::make_unique<SessionState>(graphicsImpl);
 
-                    if (!m_sessionState->System.IsInitialized())
+                    if (!m_system.IsInitialized())
```

Comment:
So the pattern is that `m_system` has the scope of the `XR` object, *but* it is still lazily initialized by the first `XRSession`?

Reply:
this seems less likely to introduce any regressions for the different platforms. The system component we are accessing prior to the xr session being created doesn't require the system to be initialized.